website/config/environment.js

```diff
@@ -30,7 +30,7 @@ module.exports = function (environment) {
 
     'ember-meta': {
       description:
-        'Helios Design System provides the building blocks to design and implement consistent, accessible, and delightful product experiences across HashiCorp.',
+        'The Helios Design System from HashiCorp provides the open source building blocks to design and implement consistent, accessible, and delightful product experiences',
```

Comment:
```suggestion
        'The Helios Design System from HashiCorp provides the open-source building blocks to design and implement consistent, accessible, and delightful product experiences.',
```

I believe "open source" is used here as an adjective and as such should be hyphenated.

Reply:
see my message in slack but we have decided as a company not to hyphenate